components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -439,14 +465,96 @@ public void saveBusinessOwner(BusinessOwner businessOwner){
+        return businessOwnerId;
     }
 
+    /**
+     * Search business owners.
+     * @param startIndex
+     * @param pageSize
+     * @param searchValue
+     * @return
+     * @throws AppManagementException
+     */
+    public List<BusinessOwner> searchBusinessOwners(int startIndex, int pageSize, String searchValue) throws
+                                                                                              AppManagementException {
+        Connection connection = null;
+        PreparedStatement statementToGetBusinessOwners = null;
+        List<BusinessOwner> businessOwnersList = new ArrayList<BusinessOwner>();
+        ResultSet businessOwnerResultSet = null;
+
+        try {
+            connection = APIMgtDBUtil.getConnection();
+            String queryToGetBusinessOwner = "SELECT * FROM BUSINESS_OWNER WHERE OWNER_NAME LIKE ? OR " +
+                    "OWNER_EMAIL LIKE ? OR OWNER_SITE LIKE ? OR OWNER_DESC LIKE ? ";
+            if (connection.getMetaData().getDriverName().contains("Oracle")) {
+                queryToGetBusinessOwner += "WHERE ROWNUM >= ? AND ROWNUM <= ?";
+            } else {
+                queryToGetBusinessOwner += "LIMIT ? , ? ";
+            }
+            statementToGetBusinessOwners = connection.prepareStatement(queryToGetBusinessOwner);
+            searchValue = "%" + searchValue + "%";
+            statementToGetBusinessOwners.setString(1, searchValue);
+            statementToGetBusinessOwners.setString(2, searchValue);
+            statementToGetBusinessOwners.setString(3, searchValue);
+            statementToGetBusinessOwners.setString(4, searchValue);
+            statementToGetBusinessOwners.setInt(5, startIndex);
+            statementToGetBusinessOwners.setInt(6, pageSize);
+            businessOwnerResultSet = statementToGetBusinessOwners.executeQuery();
+
+            while (businessOwnerResultSet.next()) {
+                BusinessOwner businessOwner = new BusinessOwner();
+                int businessOwnerId = businessOwnerResultSet.getInt("OWNER_ID");
+
+                businessOwner.setBusinessOwnerId(businessOwnerId);
+                businessOwner.setBusinessOwnerName(businessOwnerResultSet.getString("OWNER_NAME"));
+                businessOwner.setBusinessOwnerDescription(businessOwnerResultSet.getString("OWNER_DESC"));
+                businessOwner.setBusinessOwnerEmail(businessOwnerResultSet.getString("OWNER_EMAIL"));
+                businessOwner.setBusinessOwnerSite(businessOwnerResultSet.getString("OWNER_SITE"));
+                businessOwnersList.add(businessOwner);
+            }
+        } catch (SQLException e) {
+            handleException("Failed to retrieve business owner for the search value " + searchValue, e);
+        } finally {
+            APIMgtDBUtil.closeAllConnections(statementToGetBusinessOwners, connection, businessOwnerResultSet);
+        }
+        return businessOwnersList;
+    }
+
+    /**
+     * Get business owner count.
+     * @return
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5